apps/dh/app-dh/src/styles.scss

```diff
@@ -16,7 +16,7 @@
  */
 /* You can add global styles to this file, and also import other style files */
 @use "@energinet-datahub/watt";
-@import "~highlight.js/styles/github.css";
+@use "highlight.js/styles/github.css";
```

Comment:
Praise: Nice find, I has trouble finding documentation on how to do this in vite

Reply:
Thanks. It's a recommendation from Sass now to use `@use` instead of `@import`. See https://sass-lang.com/documentation/at-rules/import/

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/d0b0bf1b-db00-49b2-b3b2-f6127f9bcc31)